[MM-36879] Add additional developer setup instructions (#64)
mattermost/mattermost-app-zendesk: README.md

## Development

Mattermost Server Setup

> To allow the Mattermost server to communicate with a Zendesk cloud instance, you must expose your local mattermost instance via a tool such as ngrok.io. Set the `MM_SERVICESETTINGS_SITEURL` environment variable to this exposed URL and restart your mattermost server. You can ensure the url is correct via `System Console` > `ENVIRONMENT` > `Web Server` > `Site URL`

Reviewer: ```suggestion
To allow the Mattermost server to communicate with a Zendesk cloud instance, your Mattermost instance needs to be exposed to the internet via a tool such as [ngrok.io](https://ngrok.io). The Mattermost server then needs to be configured to use the exposed address. You can set it in the system console at `System Console` > `ENVIRONMENT` > `Web Server` > `Site URL`, or set the `MM_SERVICESETTINGS_SITEURL` environment variable. You'll need to restart your Mattermost server after you change the Site URL. You can ensure the URL is correct by clicking "Test Live URL" in the system console page mentioned above.
```

Author: Much better.  Thank you sir :)

> To allow the Mattermost server to communicate with a Zendesk cloud instance, your Mattermost instance needs to be exposed to the internet via a tool such as [ngrok.io](https://ngrok.io). The Mattermost server then needs to be configured to use the exposed address. You can set it in the system console at `System Console` > `ENVIRONMENT` > `Web Server` > `Site URL`, or set the `MM_SERVICESETTINGS_SITEURL` environment variable. You'll need to restart your Mattermost server after you change the Site URL. You can ensure the URL is correct by clicking "Test Live URL" in the system console page mentioned above.

Reviewer 2: ```suggestion
To allow the Mattermost server to communicate with a Zendesk cloud instance, your Mattermost instance needs to be exposed to the internet via a tool such as [ngrok.io](https://ngrok.io). The Mattermost server then needs to be configured to use the exposed address. You can set it via **System Console > ENVIRONMENT > Web Server > Site URL**. Then select **Test Live URL** to confirm that the URL is correct. You can also set the `MM_SERVICESETTINGS_SITEURL` environment variable. 

You'll need to restart your Mattermost server after you change the Site URL. 
```

To install Zendesk in a development environment:

`/apps debug-add-manifest --url http://localhost:4000/manifest.json`  

> `/apps install http com.mattermost.zendesk`

Author: You are correct.  An `http` install uses the URL and an `aws` install uses the pluginID.  For the install instructions, I should probably use the AWS instructions, correct?

> `/apps install http com.mattermost.zendesk`

Reviewer: For installing in a development environment (what this section describes), I suggest using the http install version. If I were to add any features to this project, I would want to use the http mode so I can be up and running quick. What value does the aws deployment give the developer that is working on the App?

> `/apps install http com.mattermost.zendesk`

Author: 👍 you're correct.  This is for development and should use `http` not `aws`.
